package.json

```diff
@@ -85,7 +85,7 @@
     },
     {
       "path": "./packages/runtime/dist/browser/index.js",
-      "limit": "313B",
+      "limit": "697B",
```

Comment:
Help me here @madou. I can decrease it a bit by inlining function body where it is being used.